sct.py

```diff
@@ -592,9 +592,12 @@ def collect_logs(test_id=None, logdir=None, backend=None, config_file=None):
 @click.option('--test-id', help='Test-id of run')
 @click.option('--test-status', help='Override test status FAILED|ABORTED')
 @click.option('--start-time', help='Override test start time')
+@click.option('--started-by', help='Default user that started the test')
 @click.option('--email-recipients', help="Send email to next recipients")
 @click.option('--logdir', help='Directory where to find testrun folder')
-def send_email(test_id=None, test_status=None, start_time=None, email_recipients=None, logdir=None):
+def send_email(test_id=None, test_status=None, start_time=None, started_by=None, email_recipients=None, logdir=None):
+    if started_by is None:
+        started_by = os.environ.get('RUN_BY_USER', None)
```

Comment:
It's better to replace it with call of `sdcm.utils.get_username.get_username()`, I think.

Reply:
agree, fixed